lib/plivo/version.rb

```diff
@@ -1,3 +1,3 @@
 module Plivo
-  VERSION = "4.39.0".freeze
+  VERSION = "4.39.1".freeze
```

Comment:
As it is not a patch, increase the minor version instead of the patch version. New version should be 4.40.0